src/Avalonia.Base/Media/DrawingGroup.cs

```diff
@@ -22,13 +22,8 @@ public class DrawingGroup : Drawing
         public static readonly StyledProperty<IBrush?> OpacityMaskProperty =
             AvaloniaProperty.Register<DrawingGroup, IBrush?>(nameof(OpacityMask));
 
-        public static readonly DirectProperty<DrawingGroup, DrawingCollection> ChildrenProperty =
-            AvaloniaProperty.RegisterDirect<DrawingGroup, DrawingCollection>(
-                nameof(Children),
-                o => o.Children,
-                (o, v) => o.Children = v);
-
-        private DrawingCollection _children = new DrawingCollection();
+        public static readonly StyledProperty<DrawingCollection> ChildrenProperty =
+            AvaloniaProperty.Register<DrawingGroup, DrawingCollection>(nameof(Children));
 
```

Comment:
In light of the comments on `Animation`, what do we think about these being `StyledProperty`? This type doesn't inherit from `StyledElement` either.

Reply:
I tried to comment on the whole block of properties but Github is only showing the changed lines.

Reply:
Personally I would focus on controls. Not drawing and animation primitives. I don't think these types of things ever really get styled anyway?

Reply:
That's what I'm asking about; the properties above this one are `StyledProperty` too.

Reply:
Given that this is going to be a multi-step process, we should just do the non-controversial control properties first IMO. We can discuss the rest later.